packages/client/lib/config.ts

```diff
@@ -288,7 +288,8 @@ export class Config {
       // Otherwise parse transports from transports option
       this.servers = parseTransports(this.transports).map((t) => {
         if (t.name === 'rlpx') {
-          const bootnodes = this.bootnodes ?? this.chainCommon.bootstrapNodes()
+          // TODO: Commented out this.chainCommon.bootstrapNodes() as its type is incompatible with the format required by RlpxServer
+          const bootnodes = this.bootnodes ?? [] // this.chainCommon.bootstrapNodes()
```

Comment:
One can't just comment out default values only because the types doesn't work any more 😜, this is obviously leading the client to not work anymore if the default bootstrap nodes are not set any more??? 😀 😀 😀

(can confirm, just tested)

Reply:
Yeah, these kind of conclusions just can't be drawn at all I guess. There is just so much of an open design space how to work around not matching types (at the end that was the "standard" way of during things in vanilla JS 😅), being it just using a subset of keys in a dict or even using an `any` cast in the consuming function.

In this case I was curious and followed it up: indeed just an [any cast](https://github.com/ethereumjs/ethereumjs-monorepo/blob/master/packages/client/lib/util/parse.ts#L42) in the consuming function together with non-matching types on the function signature. 😜 Definitely not optimal on the implementation side with (large) room for improving things. Nevertheless as some illustration how things just *are* in practice and need to be considered. 😬